src/templates/pt/2019/featured_chapters.html

```diff
@@ -0,0 +1,68 @@
+{%- set featured_chapter = ("javascript","javascript") | random %}
+{#
+  Below is the full set of chapters. If all are translated then replace above line with this one.
+  Other than add chapters to above first line as they are translated (min of two chapters so repeat if only one chapter)
+  {%- set featured_chapter = ("accessibility","caching","cdn","cms","compression","css","ecommerce","fonts","http2","javascript","markup","media","mobile-web","page-weight","performance","pwa","resource-hints","security","seo","third-parties") | random %}
+#}
+{%- if featured_chapter == "javascript" %}
+  {%- set featured_chapter_quote = "JavaScript é o recurso mais caro que enviamos aos navegadores; tendo que ser baixado, analisado, compilado e finalmente executado. Embora os navegadores tenham diminuído significativamente o tempo que leva para analisar e compilar scripts, download e execução se tornaram as etapas mais caras quando o JavaScript é processado por uma página da web." %}
+  {%- set featured_chapter_stats = {"stat1":"89%","label1":"Sites com mais código de terceiros do que primários","stat2":"83%","label2":"Sites que usam jQuery","stat3":"4.6%","label3":"Páginas iniciais usando React"} %}
```

Comment:
I've only turned on the JavaScript chapter in line 1 so that's the only one I've translated. We can translate the other chapters as we finish those translations.

@HakaCode could you review the translation here? The main quote I stole from your chapter translation but the stats labels I used Google Translate so am sure there's a few things that could be better phrased!